src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -27,25 +27,25 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = ({ isOpen, isLoading }) =
   const bestQuote = availableSwappersWithMetadata?.[0]?.quote
   const bestBuyAmountCryptoPrecision =
     bestQuote && fromBaseUnit(bestQuote.buyAmountCryptoBaseUnit, bestQuote.buyAsset.precision)
+  const bestBuyAmountCryptoPrecisionAfterSlippage = bnOrZero(bestBuyAmountCryptoPrecision)
+    .times(bn(1).minus(bnOrZero(bestQuote?.recommendedSlippage ?? DEFAULT_SLIPPAGE)))
```

Comment:
are slippage values always normalized to the same units?

Reply:
Yes - it's only calculated in one place currently (`packages/swapper/src/swappers/thorchain/getThorTradeQuote/getSlippage.ts`), though we should ensure when we begin using the THORSwap quote endpoint that the response value has the same units.